includes/Ad_Manager.php

```diff
@@ -77,15 +77,21 @@ public function print_ad_manager_tag() {
 		if ( ! $enabled || ! $slot ) {
 			return;
 		}
+
+		$application_json = [];
```

Comment:
Simplifying & fixing lint warnings (assignments need to be aligned):

```suggestion
		$configuration = [
			'ad-attributes' => [
				'type'      => 'doubleclick',
				'data-slot' => $slot,
			],
		];
```